create.sql

```diff
@@ -1261,7 +1302,7 @@ ALTER TABLE ONLY step_science
 REVOKE ALL ON SCHEMA public FROM PUBLIC;
 REVOKE ALL ON SCHEMA public FROM postgres;
 GRANT ALL ON SCHEMA public TO postgres;
-GRANT ALL ON SCHEMA public TO rnorris;
+GRANT ALL ON SCHEMA public TO swalker;
```

Comment:
It's not clear to me why this happens since the create is run as `postgres` and we never give ourselves grants.